policyengine_us/variables/gov/states/mt/tax/income/deductions/itemized/mt_itemized_deduction.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class mt_itemized_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana itemized deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf"
+        "https://leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0310/0150-0300-0210-0310.html"
+    )
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.irs.deductions
+        itm_deds = [
+            deduction
+            for deduction in p.itemized_deductions
+            if deduction not in ["salt_deduction"]
+        ]
+        filing_status = tax_unit("filing_status", period)
+        us_itm_deds_less_salt = add(tax_unit, period, itm_deds)
+        capped_property_taxes = min_(
+            add(tax_unit, period, ["real_estate_taxes"]),
+            p.itemized.salt_and_real_estate.cap[filing_status],
+        )
+
```

Comment:
we need to include [other misc. deductions](https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf) - child care expenses etc. (line 14) - probably need a separate MT variable for that

Reply:
created a variable called mt_misc_deduction